.make/Makefile.deploy.prow

```diff
@@ -21,16 +21,16 @@ build-hook: update
 
 .PHONY: build-hook-image
 build-hook-image:
-	$(DOCKER) build --build-arg BINARY=hook -t $(REGISTRY)/$(DOCKER_REPO)/hook -f $(DEPLOY_DOCKERFILE) .
-	$(DOCKER) tag $(REGISTRY)/$(DOCKER_REPO)/hook $(REGISTRY)/$(DOCKER_REPO)/hook:$(TAG)
+	$(DOCKER) build --build-arg BINARY=hook -t $(REGISTRY)/$(DOCKER_REPO)-hook -f $(DEPLOY_DOCKERFILE) .
+	$(DOCKER) tag $(REGISTRY)/$(DOCKER_REPO)-hook $(REGISTRY)/$(DOCKER_REPO)-hook:$(TAG)
```

Comment:
Why is it `-` instead of `/` now?

Reply:
Maybe we can also change `DOCKER_REPO` -> `IMG_REPO`?

Reply:
Because it changed from `prod.registry.devshift.net/osio-prod/ike-prow-plugins/test-keeper` to `quay.io/openshiftio/ike-prow-plugins-test-keeper` as per the new repositories. 

Here's the discussion -  https://chat.openshift.io/developers/pl/tqzbcjx5f3nmdg7co9fn5kay5a

Reply:
Thanks for the link. In such a case I think we should improve the naming here, as ` $(REGISTRY)/$(DOCKER_REPO)-hook` is not really correct. 

In our case this should be:

`quay.io/openshiftio/rhel-ike-prow-plugins-work-in-progress`

so 

 * `REGISTRY = quay.io`
 * `IMG_REPO = openshift.io`
*  `ike-prow-plugins-` is a prefix of the plugin image name in such case.